src/content/post/drush-make-gitignore.md

```diff
@@ -0,0 +1,52 @@
++++
+date        = "2016-04-08T01:00:00-04:00"
```

Comment:
@Pinolo please, add the following line to your front-matter:

```
author = "Pinolo"
```

so your author-block will appear in the post.